test/cpp/api/operations.cpp

```diff
@@ -55,3 +55,33 @@ TEST_F(OperationTest, Cross) {
     ASSERT_TRUE(out.allclose(exp));
   }
 }
+
+TEST_F(OperationTest, Linear_out) {
+  {
+    const auto x = torch::arange(100., 118).resize_({3, 3, 2});
+    const auto w = torch::arange(200., 206).resize_({3, 2});
```

Comment:
nitpick: I've noticed `x` is always a `3d` case?

Reply:
Will be handled by OpInfo PR